tests/unittests/Foundation/NSRecursiveLockTests.m

```diff
@@ -20,143 +20,6 @@
 #import <thread>
 #import <mutex>
 #import <condition_variable>
-#import <chrono>
-
-#import "NSRecursiveLock+Internal.h"
```

Comment:
[https://github.com/Microsoft/WinObjC/issues/688](https://github.com/Microsoft/WinObjC/issues/688)

---
In reply to: [72337799](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72337799) [](ancestors = 72337799,72325371)